src/chart-review/chart-review.component.tsx

```diff
@@ -28,6 +28,20 @@ export default function ChartReview(props: any) {
   const [selected, setSelected] = React.useState(getInitialTab());
   const [tabHistory, setTabHistory] = React.useState({});
 
+  // NAV HEADER
+  const [navHeaderOverflowed, setNavHeaderOverflowed] = React.useState(false);
```

Comment:
I think the three states here should be put into a single one using `useReducer`. They are connected and usually not set individually (e.g., once its overflowed its left, right, or both, otherwise its usually none).

Alternatively, a single state using an enum could be good:

```ts
enum HeaderOverflow {
  none = 0,
  left = 1,
  right = 2,
  both = 3,
};
```

where `HeaderOverflow.both` is just there for convenience, as it is `HeaderOverflow.left | HeaderOverflow.right`.

Reply:
sounds good